test/cli/commands.js

```diff
@@ -16,7 +14,7 @@ describe('commands', () => runOnAndOff((thing) => {
 
   it('list the commands', () => {
     return ipfs('commands').then((out) => {
-      expect(out.split('\n')).to.have.length(commandCount + 1)
+      expect(out.split('\n')).to.have.length.above(50)
```

Comment:
Please use the commandCount and not a loose assertion like `above`

Reply:
I strongly disagree.  Commands come and go.  This is a very **brittle test**. Why test for the number of commands?  All we want to test is that the commands can be obtained.

Reply:
It is not brittle if it is testing that the output of the commands call is returning the exact number of commands.

Reply:
@diasdavid I just find it odd that when I add a command, there's this little test that I don't know about that then fails.  Considering that `js-ipfs` tests take ~30 mins to run and some tests fail indeterminately, it gets overlooked.

So I will agree to disagree with you. 

Do you want me to restore the original hard coded count?